composer.json

```diff
@@ -22,6 +22,7 @@
     },
     "minimum-stability": "dev",
     "require": {
-        "squizlabs/php_codesniffer": "3.*"
+        "squizlabs/php_codesniffer": "3.*",
+        "phpunit/phpunit": "^4.0 || ^5.0 || ^6.0"
```

Comment:
I recommend to `require` phpunit as `require-dev`

Reply:
done.